app/HutchManager/Models/AgentDataSourceModel.cs

```diff
@@ -0,0 +1,16 @@
+using System.Text.Json.Serialization;
+
+namespace HutchManager.Models;
+
+public class AgentDataSource
```

Comment:
Match the filename and class name for single class files.

I don't have strong feelings about whether it has `Model` on the end, but ideally we should be consistent.

Reply:
Also noticed this model doesn't represent an Agent DataSource; just an Agent (which happens to have a list of associated DataSource names), so just call it `AgentModel` or `Agent`, or if it's for a specific purpose you could put that in the name (maybe `AgentSummary`?)

Reply:
I think AgentSummary suits the purpose.